mu-plugins/10up-plugin/includes/core.php

```diff
@@ -18,21 +18,17 @@
  * @return void
  */
 function setup() {
-	$n = function ( $function ) {
-		return __NAMESPACE__ . "\\$function";
-	};
-
-	add_action( 'init', $n( 'i18n' ) );
-	add_action( 'init', $n( 'init' ), apply_filters( 'tenup_plugin_init_priority', 8 ) );
-	add_action( 'wp_enqueue_scripts', $n( 'scripts' ) );
-	add_action( 'wp_enqueue_scripts', $n( 'styles' ) );
-	add_action( 'admin_enqueue_scripts', $n( 'admin_scripts' ) );
-	add_action( 'admin_enqueue_scripts', $n( 'admin_styles' ) );
+	add_action( 'init', __NAMESPACE__ . '\\i18n' );
+	add_action( 'init', __NAMESPACE__ . '\\init', apply_filters( 'tenup_plugin_init_priority', 8 ) );
+	add_action( 'wp_enqueue_scripts', __NAMESPACE__ . '\\scripts' );
+	add_action( 'wp_enqueue_scripts', __NAMESPACE__ . '\\styles' );
+	add_action( 'admin_enqueue_scripts', __NAMESPACE__ . '\\admin_scripts' );
+	add_action( 'admin_enqueue_scripts', __NAMESPACE__ . '\\admin_styles' );
```

Comment:
```suggestion
	add_action( 'init', 'TenUpPlugin\Core\i18n' );
	add_action( 'init', 'TenUpPlugin\Core\init', apply_filters( 'tenup_plugin_init_priority', 8 ) );
	add_action( 'wp_enqueue_scripts', 'TenUpPlugin\Core\scripts' );
	add_action( 'wp_enqueue_scripts', 'TenUpPlugin\Core\styles' );
	add_action( 'admin_enqueue_scripts', 'TenUpPlugin\Core\admin_scripts' );
	add_action( 'admin_enqueue_scripts', 'TenUpPlugin\Core\admin_styles' );
```

@darylldoyle my preference based on a long conversation with @tobeycodes in https://github.com/10up/ui-kit-scaffold/pull/8#discussion_r1504348038 would be to use the full function including the namespace directly as this has much better IDE integration